### PR TITLE
#2tdmk1f

### PR DESCRIPTION
**Pull request by Kirosana15**

### CODE REVIEW

File: src/auth/test/mock/auth.service.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { UserCredentialsDto } from '../../../user/dto/user.credentials.dto';
 import { generateMockToken, generateUserFromDb } from '../../../user/test/mock/user.model.mock';
-import { RefreshTokenDto } from '../../dto/tokens.dto';
+
```
Comment (PrzerostNadSens) [inline src/auth/test/mock/auth.service.ts]: you can remove this space here

File: src/dto/queries.dto.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import { IsNumber, IsOptional } from 'class-validator';
+
```
Comment (PrzerostNadSens) [inline src/dto/queries.dto.ts]: same as before

File: src/user/test/all-get-user.controller.spec.ts [hunk 3]
```diff
@@ -0,0 +1,74 @@
+import { HttpStatus, ValidationPipe } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+
+import { NestApplication } from '@nestjs/core';
+import { UserController } from '../user.controller';
+import { UserInfoDto } from '../dto/user.dto';
+import { UserService } from '../user.service';
+import { mockUserService } from './mock/user.service.mock';
+import request from 'supertest';
+
```
Comment (PrzerostNadSens) [inline src/user/test/all-get-user.controller.spec.ts:10]: I will import it that way 

```
import { HttpStatus, ValidationPipe } from '@nestjs/common';
import { Test, TestingModule } from '@nestjs/testing';
import { NestApplication } from '@nestjs/core';
import request from 'supertest';

import { UserController } from '../user.controller';
import { UserService } from '../user.service';
import { UserInfoDto } from '../dto/user.dto';
import { mockUserService } from './mock/user.service.mock';
```

first libraries 
then space 
then import from the same folder next to each other

File: src/user/test/get-all-user.service.spec.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { MongooseModule, getModelToken } from '@nestjs/mongoose';
+import { Test, TestingModule } from '@nestjs/testing';
+import { User, UserDocument, UserSchema } from '../user.schema';
+import { closeConnections, rootMongooseTestModule } from './mock/db.mock';
+
+import { Model } from 'mongoose';
+import { UserService } from '../user.service';
+import { generateCredentialsList } from './mock/user.model.mock';
+
```
Comment (PrzerostNadSens) [inline src/user/test/get-all-user.service.spec.ts:9]: same as before

File: src/user/test/all-get-user.controller.spec.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
+import { HttpStatus, ValidationPipe } from '@nestjs/common';
```
Comment (PrzerostNadSens) [inline src/user/test/all-get-user.controller.spec.ts:1]: maybe better change name of the file to `get-all-user.controller...`?

**Review by PrzerostNadSens — APPROVED** [review]
please pay attention to the order of imports in all files

File: src/user/test/all-get-user.controller.spec.ts (hunk 5 above)
Comment (Kirosana15) [reply src/user/test/all-get-user.controller.spec.ts:1]: In this case it looks weird, but I'm using the convention of endpoint-httpmethod-module.controller...

**Pull request merged**